test/suites/tuned/workload-partitioning.robot

```diff
@@ -62,7 +61,7 @@ Teardown For Workload Partitioning
     ...    ${SYSTEMD_MICROSHIFT_DROPIN}
     ...    ${CRIO_CONFIG_DROPIN}
     Systemctl Daemon Reload
-    Restore Default MicroShift Config
+    Remove Kubelet Config Drop In
```

Comment:
This is only called once, why are the other robot files not getting the same keyword?

Reply:
good point